bears/general/LineCountBear.py

```diff
@@ -39,3 +60,12 @@ def run(self, filename, file, max_lines_per_file: int,
                                  extra=file_length-max_lines_per_file)),
                 severity=RESULT_SEVERITY.NORMAL,
                 file=filename)
+        if file_length < min_lines_per_file:
+            yield Result.from_values(
+                origin=self,
+                message=('This file had {count} lines, which is {missing} '
+                         'lines fewer than required.'
+                         .format(count=file_length,
+                                 missing=min_lines_per_file - file_length)),
```

Comment:
I would include in the message the given min-lines-per-file setting, so the user remembers what he has passed. Consider you have a few files violating this min-lines-per-file, It's a bit confusing if both numbers change in every result (because all flagged files have a different number of lines)